http_logs/challenges/default.json

```diff
@@ -220,4 +220,52 @@
           "clients": 1
         }
       ]
-    }
\ No newline at end of file
+    },
+    {
+      "name": "update",
+      "schedule": [
+        {
+          "operation": "delete-index"
+        },
+        {
+          "operation": {
+            "operation-type": "create-index",
+            "settings": {%- if index_settings is defined %} {{index_settings | tojson}} {%- else %} {
+              "index.number_of_shards": {{number_of_shards | default(1)}},
+              "index.number_of_replicas": {{number_of_replicas | default(0)}},
+              "index.store.type": "{{store_type | default('mmapfs')}}"
```

Comment:
Should we remove this for the time being? I don't think we need this parameter for every track.

Reply:
As discussed, I'll leave this here as we removed it when we also remove it from nyc_taxis.